README.md

```diff
@@ -4,7 +4,18 @@ Gatsby starter with TypeScript, Emotion, Contentful, and Remark.
 
 ## 🚀 Quick start
 
-1.  **Install.**
+0.  **Install Yarn (if Yarn Package Manager is not available on your machine).**
+
+Following below is the command you'll need to use if you've installed Node.js with `nvm`.
+
+    ```sh
+    # install yarn
+    brew install yarn --ignore-dependencies
```

Comment:
this is a mac specific solution. Should we perhaps just include the below text pointing to installation guide?

Reply:
Totally 👍🏽